src/httpserver.cpp

```diff
@@ -438,6 +430,8 @@ bool InitHTTPServer()
     workQueue = new WorkQueue<HTTPClosure>(workQueueDepth);
     eventBase = base;
     eventHTTP = http;
+    base_ctr.release();
+    http_ctr.release();
```

Comment:
Because `eventBase`/`eventHTTP` are global and need them intact. If I don't release they'd be deleted on function return.